<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>queryroom</class>
 <widget class="QDialog" name="queryroom">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>关键字</string>
   </property>
  </widget>
  <widget class="QPushButton" name="checkButton_4">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="upButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>190</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>升序</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="downButton_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>190</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>降序</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>80</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>排序方式</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="orderidButton_3">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>110</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>房间号</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="ordertypeButton_4">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>140</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>房间类型</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="orderpriceButton_5">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>170</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>房间价格</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
